src/email-address.ts

```diff
@@ -85,7 +87,5 @@ export class EmailAddress {
 }
 
 export function emailIsValid(email: string): boolean {
-  return /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>().,;\s@"]+\.{0,1})+([^<>().,;:\s@"]{2,}|[\d.]+))$/.test(
-    email
-  )
+  return EmailValidator.validate(email)
```

Comment:
Are we sure that the security issue is removed using this library? for what I see in its implementation, it also performs a regexp test against a very complex pattern: 
https://github.com/manishsaraan/email-validator/blob/113194f436418b4d868d3950d9a50700846eced7/index.js#L30

Reply:
hi @yeraydavidrodriguez they [validate the length of the parts of the email](https://github.com/manishsaraan/email-validator/blob/113194f436418b4d868d3950d9a50700846eced7/index.js#L20-L22) before using the regex. I suppose the time to validate an email with a maximum length of 320 chars should not be too long.

And it seems [other people have already opened an issue](https://github.com/manishsaraan/email-validator/issues/46) and it seems to be safe.

Reply:
This [other package](https://github.com/spamscanner/email-regex-safe) looks better since explicitly uses [node-re2](https://github.com/uhop/node-re2) to check vulneravilities.

It also allows more cases that are false negatives in the current package.

I'm going to give it a try.